### PR TITLE
feat: document Operate and Tasklist backup and restore process

### LINKED ISSUE #1151: Document hot backup for Operate and Tasklist
- [ ] document how to use Hot backup API of Operate and Tasklist
- [ ] adjust documentation of [Actuator settings](https://docs.camunda.io/docs/self-managed/operate-deployment/operate-configuration/#monitoring-operate)

### PR DESCRIPTION
**Pull request by sdorokhova**
## What is the purpose of the change

- Document Operate and Tasklist backup and restore process
- clean up Actuator settings docs.

closes #1151

## Are there related marketing activities

_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._

## When should this change go live?

8.1.0

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by sdorokhova** [comment]
@christinaausley I asked you for review to check the wording. I hope you're the right person for that)

**Review by ralfpuchert — APPROVED** [review]

File: sidebars.js [hunk 1]
```diff
@@ -977,7 +977,10 @@ module.exports = {
       "Zeebe Gateway": ["self-managed/zeebe-gateway-deployment/zeebe-gateway"],
     },
     {
-      "Backup & Restore": ["self-managed/backup-restore/backup-and-restore"],
+      "Backup & Restore": [
+        "self-managed/backup-restore/backup-and-restore",
+        "self-managed/backup-restore/operate-tasklist-backup",
+      ],
```
Comment (pepopowitz) [inline sidebars.js:983]: Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1540.

**Review by pepopowitz — DISMISSED** [review]
Additional changes are required to keep the sidebars from drifting in two separate areas of the site. 

See #1116 for more details on why these changes are needed, or let me know if I can further clarify!

File: sidebars.js (hunk 1 above)
Comment (akeller) [reply sidebars.js:983]: I'll take care of this during my review.

**Review by akeller — APPROVED** [review]

**Pull request merged**